docs/tools/debug/memap.md

```diff
@@ -72,4 +72,4 @@ There are other entries that require a bit of clarification:
 
 ## Current support
 
-We have tested memap on Windows 7, Linux and macOS. The `GCC_ARM` (GNU Arm Embedded Toolchain), Arm Compiler 5, Arm Compiler 6 and IAR toolchains generate memory map files.
+We have tested memap on Windows 7, Linux and macOS. The `GCC_ARM` (GNU Arm Embedded Toolchain), Arm Compiler 6 and IAR toolchains generate memory map files.
```

Comment:
Which Linux distro? Which OS X version?

Reply:
I don't know. I'll try to find out who did the testing.

Reply:
Thanks as it may not work with latest versions of OS X or with another Linux distribution.